### PR TITLE
Fixes to Dockerfile

### PR DESCRIPTION
**Pull request by omus**
I've been using the `Dockerfile` again and discovered a problem and some outdated instructions.

### CODE REVIEW

**Comment by omus** [comment]
Appears when building on the CI Julia cares about the empty package structure:
```
#25 [deps 4/5] RUN --mount=type=cache,sharing=locked,target=/mnt/julia-cache,uid=1000,gid=100     rm -rf /home/ray/.julia/dev/Ray &&     mkdir -p $(dirname /home/ray/.julia/dev/Ray) &&     mv /tmp/Ray.jl /home/ray/.julia/dev/Ray &&     julia --project=/home/ray/.julia/dev/Ray -e 'using Pkg; Pkg.Registry.update(); Pkg.instantiate(); Pkg.build(); Pkg.precompile(strict=true, timing=true)'
#25 1.190     Updating registry at `~/.julia/registries/General.toml`
#25 1.470     Updating registry at `~/.julia/registries/General.toml`
#25 3.186 ERROR: expected the file `src/Ray.jl` to exist for package `Ray` at `/mnt/julia-cache/dev/Ray`
```

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # syntax=docker/dockerfile:1
 # Dockerfile is currently only for x86_64
 
+# ```sh
+# docker build ray:2.5.1-julia1.9.3-rayjl$(git rev-parse --short HEAD) .
+# ```
```
Comment (glennmoy) [inline Dockerfile:6]: not sure if you want to remove these lines first

Comment (omus) [reply Dockerfile:6]: I intended to add that as a example of how someone may want to tag the package. However, now that you brought this up I see I forgot to add the `-t` flag. I'll add that and a short explanation.

Comment (omus) [inline Dockerfile:6]:
```suggestion
# Example of how to build this Docker image including a recommended tagging structure:
# ```sh
# docker build -t ray:2.5.1-julia1.9.3-rayjl$(git rev-parse --short HEAD) .
# ```
```

**Pull request merged**